### ISSUE TITLE
Publish this in the marketplace

### ISSUE THREAD

**Issue by Markos-Th09** [ClueLang/Clue-for-VSCode#1]

**Comment by Maiori44** [comment]
I'm not sure if it's ready yet, but I'll see

**Comment by Markos-Th09** [comment]
> I'm not sure if it's ready yet, but I'll see

Publish it is a preview version

**Comment by Markos-Th09** [comment]
Or at least release vsix to make it easier to install

**Comment by Maiori44** [comment]
> Publish it is a preview version
A preview version sounds like a great idea
I think I'll do it soon (maybe once Clue 2.2 is out)

**Comment by Markos-Th09** [comment]
The most important thing to make clue a usable would be making a language server (probably modifying and existing Lua language server)

**Comment by Maiori44** [comment]
> The most important thing to make clue a usable would be making a language server (probably modifying and existing Lua language server)

I agree, I tried looking at Lua's language server a while ago but I couldn't figure it out
I'm not sure if I would be able to modify/make another myself

**Comment by Markos-Th09** [comment]
> > The most important thing to make clue a usable would be making a language server (probably modifying and existing Lua language server)
> 
> I agree, I tried looking at Lua's language server a while ago but I couldn't figure it out I'm not sure if I would be able to modify/make another alone

I took a look at the code and understood what had to be changed (the parser and the ast) though I wasn't able to understand how it should be changed either

**Comment by Maiori44** [comment]
> > > The most important thing to make clue a usable would be making a language server (probably modifying and existing Lua language server)
> > 
> > 
> > I agree, I tried looking at Lua's language server a while ago but I couldn't figure it out I'm not sure if I would be able to modify/make another alone
> 
> I took a look at the code and understood what had to be changed (the parser and the ast) though I wasn't able to understand how it should be changed either

hm, maybe we could ask them for help, not sure tho
